[2024-08-30T09:41:45.907+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-30T09:41:45.926+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v01.get_name manual__2024-08-30T09:41:29.063387+00:00 [queued]>
[2024-08-30T09:41:45.935+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v01.get_name manual__2024-08-30T09:41:29.063387+00:00 [queued]>
[2024-08-30T09:41:45.936+0000] {taskinstance.py:2856} INFO - Starting attempt 2 of 7
[2024-08-30T09:41:45.949+0000] {taskinstance.py:2879} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-08-30 09:41:29.063387+00:00
[2024-08-30T09:41:45.957+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=6692) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-30T09:41:45.959+0000] {standard_task_runner.py:72} INFO - Started process 6694 to run task
[2024-08-30T09:41:45.958+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v01', 'get_name', 'manual__2024-08-30T09:41:29.063387+00:00', '--job-id', '334', '--raw', '--subdir', 'DAGS_FOLDER/taskflow_api_dag.py', '--cfg-path', '/tmp/tmpvnso75g2']
[2024-08-30T09:41:45.959+0000] {standard_task_runner.py:105} INFO - Job 334: Subtask get_name
[2024-08-30T09:41:46.005+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v01.get_name manual__2024-08-30T09:41:29.063387+00:00 [running]> on host efc48c282cbe
[2024-08-30T09:41:46.093+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v01' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T09:41:29.063387+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-30T09:41:29.063387+00:00'
[2024-08-30T09:41:46.094+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-30T09:41:46.096+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2024-08-30T09:41:46.116+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-30T09:41:46.116+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v01, task_id=get_name, run_id=manual__2024-08-30T09:41:29.063387+00:00, execution_date=20240830T094129, start_date=20240830T094145, end_date=20240830T094146
[2024-08-30T09:41:46.174+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-30T09:41:46.190+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-30T09:41:46.193+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
